src/clj/puppetlabs/services/jruby_pool_manager/jruby_schemas.clj

```diff
@@ -59,14 +59,18 @@
         values are `:jit`, `:force`, and `:off`.  Defaults to `:off`.
 
     * :max-active-instances - The maximum number of JRubyInstances that
-        will be pooled."
+        will be pooled.
+
+    * :environment-vars - A map of whitelisted environment variables and their values to be
+        whitelisted and visible to any Ruby code."
   {:ruby-load-path [schema/Str]
    :gem-home schema/Str
    :compile-mode SupportedJRubyCompileModes
    :borrow-timeout schema/Int
    :max-active-instances schema/Int
    :max-borrows-per-instance schema/Int
-   :lifecycle LifecycleFns})
+   :lifecycle LifecycleFns
+   :environment-vars {schema/Str schema/Str}})
```

Comment:
Sorry, I don't think I explained that clearly. I think that when passed into JRuby here they need to be strings. We should just double check that when we read off the config file that these can be strings, otherwise we'll need to convert what we get from the config file.